### PR TITLE
Fix issue #1, #2

### PR DESCRIPTION
**Pull request by teraui41**
Thank you for your contribution to the develop branch of **nuxt-project-testing** repo.
 
Before submitting this PR, please make sure:

- [ ] Title should start with `Fix`, `Add`, `Modify`, `Setup`, `Implement`, etc...
- [ ] Code builds cleanly without any error or warning
- [ ] Methods and variables are using the approved terminology
- [ ] Test specs are added or updated accordingly
- [ ] Link this PR to a related issue

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -4,7 +4,7 @@
   "description": "for interview",
   "main": "index.ts",
   "scripts": {
-    "test": "nuxt build && TZ=Asia/Taipei jest",
```
Comment (echoulen) [inline package.json:7]: should not remove `TZ=Asia/Taipei`

File: src/components/TreeComponent/TreeItem.vue [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+      "level",
+      "children",
+    ],
+    data() {
+      return {
+        msg: "Hello",
+      };
+    },
+    computed: {
+      hasChildren: function() {
```
Comment (echoulen) [inline src/components/TreeComponent/TreeItem.vue:31]: `hasChildren() { ... } ` is better

File: tests/pages/index.spec.ts [hunk 3]
```diff
@@ -1,9 +1,73 @@
-import {mount} from "@vue/test-utils";
+import { mount, shallowMount } from "@vue/test-utils";
+import {mockPermissions} from "../../dummies/mockPermissions";
 import Index from "../../src/pages/index.vue";
+import { findLabel } from "../../src/utils/findLabel";
```
Comment (echoulen) [inline tests/pages/index.spec.ts:4]: wired format
```
import {mockPermissions} from ...
import { findLabel } from ...
```

**Review by nelsliu9121 — CHANGES_REQUESTED** [review]
幾個事情請在開發前先確認：

1. 檢查專案的書寫設定
  a. 檔案請以空白行結尾
  b. `import` 大括號不應該有空白間隔
  c. 不在物件宣告使用 `function`
  d. 縮排、空格應比照原設定（可依靠 `tslint`、`prettier` 協助）
2. 不任意修改對外 port 編號（或是不加入 commit）
3. 檔案取名應符合專案的習慣 （`TreeComponent/index.vue` 不合習慣）
4. 發出 Pull request 應該要檢查的項目也請一一確認勾選

**Comment by teraui41** [comment]
謝謝！這些開發習慣未來我會多加注意，也謝謝您們的指教，受益良多．

**Pull request closed without merging**